README.md

```diff
@@ -215,7 +225,7 @@ opentok.deleteArchive(archiveId);
 You can also get a list of all the Archives you've created (up to 1000) with your API Key. This is
 done using a `com.opentok.OpenTok` instance's `listArchives(int offset, int count)` method. You may optionally
 paginate the Archives you receive using the offset and count parameters. This will return a
-`List<Archive>` type.
+`List<Archive>` type. If the offset or count is negative or if the count is greater than 1000 an `InvalidArgumentException` is thrown.
```

Comment:
Please change to:

An `InvalidArgumentException` will be thrown if the offset or count are either negative or if the count is greater than 1000.

Reply:
changed to An `InvalidArgumentException` will be thrown if the offset or count are negative 
or if the count is greater than 1000.